### ISSUE TITLE
`lhapdf` grids with `evolven3fit_new` are broken

### ISSUE THREAD

**Issue by giacomomagni** [NNPDF/nnpdf#1704]
While running the Marco Bonvini's code for gluon fusion with a new fit I found this: 

```
LHAPDF 6.5.3 loading /usr/local/Cellar/lhapdf/6.5.3/share/LHAPDF/230321-gm-n3lo-dis_th_iterated/230321-gm-n3lo-dis_th_iterated_0000.dat
230321-gm-n3lo-dis_th_iterated PDF set, member #0, version 1
libc++abi: terminating with uncaught exception of type LHAPDF::MetadataError: AlphaS value and Q interpolation arrays are differently sized
zsh: abort      ./ggHiggs -p 230321-gm-n3lo-dis_th_iterated 
```

and indeed checking the info file I see two suspicious things: 
1. `AlphaS_Qs` seems to be squared
2.  Its length is different from the number of `AlphaS_Vals` provided.  

I attach here the `.info` file:

```
AlphaS_MZ: 0.118
AlphaS_OrderQCD: 4
AlphaS_Qs: [2.7224999999999997, 3.19493746374544, 3.77488100476809, 4.491749966750009, 5.384302568099999, 6.50400152667136, 7.91973571100625, 9.72449464910884, 12.044908183506251, 15.05498278164001, 18.996100349542562, 24.2064, 30.795420826788842, 39.560894680923035, 51.33820947571969, 67.32672982919689, 89.26706223637505, 119.71375253888401, 162.459802224784, 223.20545256384398, 310.625538183184, 438.09483041122496, 626.515817968804, 909.018158405184, 1338.8849614637293, 2003.1247784635239, 3046.079374924804, 4711.1668074436, 7415.951849678241, 11889.553678992901, 19428.805462562497, 32384.935751422498, 55106.71740324, 95805.9980071936, 170327.3319815824, 309935.61072033213, 577779.2979122024, 1104536.67973636, 2167541.8518547597, 4371024.81744016, 9067872.884422809, 19374523.602830008, 42685208.41230725, 97091828.80054595, 228300435.22899598, 555689440.640356, 1402054409.096289, 3672277584.4624, 10000000000.0]
AlphaS_Type: ipol
AlphaS_Vals: [0.420071, 0.389564, 0.361942, 0.336844, 0.313967, 0.293052, 0.273881, 0.256264, 0.240039, 0.225066, 0.211221, 0.198396, 0.186496, 0.175439, 0.16515, 0.155563, 0.146619, 0.138266, 0.130456, 0.123146, 0.116299, 0.109879, 0.103854, 0.0981964, 0.0928794, 0.0878793, 0.0831741, 0.0787438, 0.07457, 0.0706356, 0.0669252, 0.0634242, 0.0601195, 0.0569986, 0.0540502]
Authors: ''
DataVersion: 1
ErrorType: replicas
FlavorScheme: variable
Flavors: [-6, -5, -4, -3, -2, -1, 1, 2, 3, 4, 5, 6, 21, 22]
ForcePositive: 0
Format: lhagrid1
MBottom: 4.92
MCharm: 1.51
MDown: 0.0
MStrange: 0.0
MTop: 172.5
MUp: 0.0
MZ: 91.2
NumFlavors: 14
NumMembers: 101
OrderQCD: 4
Particle: 2212
QMax: 100000.0
QMin: 1.65
Reference: null
SetDesc: Evolved PDF from 1.65 GeV
SetIndex: 0
XMax: 1.0
XMin: 1.0e-09
```

checking with @enocera we found that this was visible also in `230311-ern-001` and maybe in other new fits.
We suspect this is due to a bug in `evolven3fit_new` (or maybe even `ekobox` where the info file is generated !? )
@andreab1997 @scarlehoff

**Comment by andreab1997** [comment]
I can try to investigate but I believe that, at least for the @enocera fit, he was using the old version (so with eko==0.10) so maybe that problem is solved now. I will investigate anyway.

**Comment by giacomomagni** [comment]
Also are we sure `AlphaS_OrderQCD` and `OrderQCD ` should read 4 for the N3LO fit and not 3 ?

**Comment by giacomomagni** [comment]
See [issue in Ekobox](https://github.com/NNPDF/eko/issues/228)

**Comment by andreab1997** [comment]
In the master version of `eko` now there is the updated version of `ekobox` that is able to dump correctly the alpha_values and alpha_Q to the info file.

**Comment by giacomomagni** [comment]
Thank you very much @andreab1997. I'll then merge master into my eko branch.
Did you fix also `AlphaS_OrderQCD` and `OrderQCD` ??

**Comment by RoyStegeman** [comment]
> But are these informations used at all?

The `.info` files are used by lhapdf, for example the theory settings can be used when calculating alphas. Probably not so interesting for us, but these settings should agree with what was done during the fit for sets that are shared outside the collaboration.

**Comment by andreab1997** [comment]
> > But are these informations used at all?
> 
> The `.info` files are used by lhapdf, for example the theory settings can be used when calculating alphas. Probably not so interesting for us, but these settings should agree with what was done during the fit for sets that are shared outside the collaboration.

Yes I did not want to ignore them, I was just curious :)

**Comment by RoyStegeman** [comment]
That's a good enough reason to ask :). From the original message I assume that @enocera is aware of this, which is good since it is indeed important that for the sets that are made public on the nnpdf website this is fixed.

**Comment by scarlehoff** [comment]
> EDIT: just to be precise, this issue has been solved in eko

Which version of eko?

The current NNPDF master uses this tag: https://github.com/NNPDF/nnpdf/actions/runs/4912594595/jobs/8771786219#step:6:718

is the issue solved with that?

**Comment by andreab1997** [comment]
> > EDIT: just to be precise, this issue has been solved in eko
> 
> Which version of eko?
> 
> The current NNPDF master uses this tag: https://github.com/NNPDF/nnpdf/actions/runs/4912594595/jobs/8771786219#step:6:718
> 
> is the issue solved with that?

No the tag in which the bug is solved is `0.12.2`

**Comment by scarlehoff** [comment]
The tag in the line I linked (or that I wanted to link, maybe I failed) is the following:

    eko:                                  0.12.2-pyhd8ed1ab_0                  conda-forge                          

is that correct? (I think it is the last tag in eko... but just to make sure before closing this issue)
